Scrapers/VenueExplorer.py

```diff
@@ -1 +1,130 @@
-# TODO: Write scraping script here
\ No newline at end of file
+from bs4 import BeautifulSoup
+from requests import get
+import VenueClass
+import json
+import os
+
+allVenues = []  # Array to store all Venues of type VenueClass
+
+def extractVenue(venue):
+
+    # extract title
+    title_html = venue.find('div', class_='event-name')
+    title = ''
+    if title_html:
+        title = title_html.text
```

Comment:
I noticed that all the title have \n

Maybe can replace line 15 with **title=title_html.text.replace("\n","").strip()** to remove the \n characters and extra spacing?